app/controllers/circles_controller.rb

```diff
@@ -1,8 +1,40 @@
 class CirclesController < ApplicationController
+  layout "users"
+  before_action :current_user
 
   def index
-    @circles = Circle.includes(:creator)
+    @circles = Circle.includes(:creator, :submissions)
     render "_index"
   end
 
+  def new
+    @circle = @current_user.circles.new
+  end
+
+  def create
+    @circle = @current_user.circles.new(circle_params)
+    if @circle.save
+      flash[:success] = "Sweet! Circle successfully created"
+      redirect_to circles_path
+    else
+      flash[:error] = ["Sorry, something went wrong. Please try again",
+                      @circle.errors.full_messages.to_sentence]
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.